scripts/quests/outlands/You_Call_That_a_Knife.lua

```diff
@@ -0,0 +1,211 @@
+-----------------------------------
+-- You Call That a Knife
+-----------------------------------
+-- Log ID: 5, Quest ID: 6
+-- Mhebi Juhbily : !pos 40 -11 -159 250
+-- Vah Keshura   : !pos 30 -8.5 -105 250
+-- Chef Nonberry : !pos -136 0.0 -91 159
+-- Pula Rhatti   : !pos  -18.5 -4 -38 250
+-----------------------------------
+local templeID = zones[xi.zone.TEMPLE_OF_UGGALEPIH]
+-----------------------------------
+
+---@type TQuest
+local quest = Quest:new(xi.questLog.OUTLANDS, xi.quest.id.outlands.YOU_CALL_THAT_A_KNIFE)
+
+quest.reward =
+{
+    fame = 75,
+    fameArea = xi.fameArea.WINDURST,
+    gil = 7200,
+    title = xi.title.YA_DONE_GOOD
+}
+
+local function isFightInProgress()
+    return GetMobByID(templeID.mob.COOK_OFFSET):isAlive() or
+        GetMobByID(templeID.mob.COOK_OFFSET + 1):isAlive() or
+        GetMobByID(templeID.mob.COOK_OFFSET + 2):isAlive() or
+        GetMobByID(templeID.mob.COOK_OFFSET + 3):isAlive()
+end
+
+local function spawnCooks(player, npc)
+    if not isFightInProgress() then
```

Comment:
popFromQM makes this check
![image](https://github.com/user-attachments/assets/0ef517ee-9aeb-4913-93a0-980026aa7e64)

Reply:
I got distracted by the `QM` part 😅
Appreciate the review here! Removed the other bits as they're no longer necessary

Reply:
Awesome! Thanks for the quick responses here.